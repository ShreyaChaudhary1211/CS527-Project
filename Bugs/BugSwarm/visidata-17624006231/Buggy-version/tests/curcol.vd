sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/sample.tsv	o	
sample	Region		addcol-expr	curcol.casefold()	=	create new column from Python expression, with column names as variables
sample	Item		addcol-expr	curcol.casefold()	=	create new column from Python expression, with column names as variables
sample	curcol.casefold()		rename-col	Region_col	^	edit name of current column
sample	curcol.casefold()		rename-col	Item_col	^	edit name of current column
