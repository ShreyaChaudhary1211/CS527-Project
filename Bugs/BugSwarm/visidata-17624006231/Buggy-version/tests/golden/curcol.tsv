OrderDate	Region	Region_col	Rep	Item	Item_col	Units	Unit_Cost	Total
2016-01-06	East	east	Jones	Pencil	pencil	95	1.99	189.05
2016-01-23	Central	central	Kivell	Binder	binder	50	19.99	999.50
2016-02-09	Central	central	Jardine	Pencil	pencil	36	4.99	179.64
2016-02-26	Central	central	Gill	Pen	pen	27	19.99	539.73
2016-03-15	West	west	Sorvino	Pencil	pencil	56	2.99	167.44
2016-04-01	East	east	Jones	Binder	binder	60	4.99	299.40
2016-04-18	Central	central	Andrews	Pencil	pencil	75	1.99	149.25
2016-05-05	Central	central	Jardine	Pencil	pencil	90	4.99	449.10
2016-05-22	West	west	Thompson	Pencil	pencil	32	1.99	63.68
2016-06-08	East	east	Jones	Binder	binder	60	8.99	539.40
2016-06-25	Central	central	Morgan	Pencil	pencil	90	4.99	449.10
2016-07-12	East	east	Howard	Binder	binder	29	1.99	57.71
2016-07-29	East	east	Parent	Binder	binder	81	19.99	1619.19
2016-08-15	East	east	Jones	Pencil	pencil	35	4.99	174.65
2016-09-01	Central	central	Smith	Desk	desk	2	125.00	250.00
2016-09-18	East	east	Jones	Pen Set	pen set	16	15.99	255.84
2016-10-05	Central	central	Morgan	Binder	binder	28	8.99	251.72
2016-10-22	East	east	Jones	Pen	pen	64	8.99	575.36
2016-11-08	East	east	Parent	Pen	pen	15	19.99	299.85
2016-11-25	Central	central	Kivell	Pen Set	pen set	96	4.99	479.04
2016-12-12	Central	central	Smith	Pencil	pencil	67	1.29	86.43
2016-12-29	East	east	Parent	Pen Set	pen set	74	15.99	1183.26
2017-01-15	Central	central	Gill	Binder	binder	46	8.99	413.54
2017-02-01	Central	central	Smith	Binder	binder	87	15.00	1305.00
2017-02-18	East	east	Jones	Binder	binder	4	4.99	19.96
2017-03-07	West	west	Sorvino	Binder	binder	7	19.99	139.93
2017-03-24	Central	central	Jardine	Pen Set	pen set	50	4.99	249.50
2017-04-10	Central	central	Andrews	Pencil	pencil	66	1.99	131.34
2017-04-27	East	east	Howard	Pen	pen	96	4.99	479.04
2017-05-14	Central	central	Gill	Pencil	pencil	53	1.29	68.37
2017-05-31	Central	central	Gill	Binder	binder	80	8.99	719.20
2017-06-17	Central	central	Kivell	Desk	desk	5	125.00	625.00
2017-07-04	East	east	Jones	Pen Set	pen set	62	4.99	309.38
2017-07-21	Central	central	Morgan	Pen Set	pen set	55	12.49	686.95
2017-08-07	Central	central	Kivell	Pen Set	pen set	42	23.95	1005.90
2017-08-24	West	west	Sorvino	Desk	desk	3	275.00	825.00
2017-09-10	Central	central	Gill	Pencil	pencil	7	1.29	9.03
2017-09-27	West	west	Sorvino	Pen	pen	76	1.99	151.24
2017-10-14	West	west	Thompson	Binder	binder	57	19.99	1139.43
2017-10-31	Central	central	Andrews	Pencil	pencil	14	1.29	18.06
2017-11-17	Central	central	Jardine	Binder	binder	11	4.99	54.89
2017-12-04	Central	central	Jardine	Binder	binder	94	19.99	1879.06
2017-12-21	Central	central	Andrews	Binder	binder	28	4.99	139.72
